plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/extractor/JenkinsfileVisitor.java

```diff
@@ -36,67 +43,198 @@ public J.VariableDeclarations visitVariableDeclarations(J.VariableDeclarations v
         return variableDeclarations;
     }
 
+    @Override
+    public J visitMapEntry(G.MapEntry mapEntry, PluginMetadata pluginMetadata) {
```

Comment:
I'm really not proud of this code.

We should use different visit callback and use cursor to get information higher on the tree. Not going down

But I least now I have test to validation behavior and will improve in the future